pkg/trait/jolokia.go

```diff
@@ -63,7 +63,7 @@ func (t *jolokiaTrait) Apply(e *Environment) error {
 		}
 
 		// TODO: We may want to make the Jolokia version configurable
```

Comment:
nit: remove this TODO, also from main if it's still there (a follow up PR would be fine)